pylxd/container.py

```diff
@@ -70,7 +70,7 @@ def create(cls, client, config, wait=False):
 
     def __init__(self, **kwargs):
         super(Container, self).__init__()
-        for key, value in kwargs.iteritems():
+        for key, value in kwargs.items():
```

Comment:
Yeah, that's why I left that comment in the pull request.

There are several similar iteritems() uses throughout your codebase. Since I don't use python2 I can't really make sure that the fix is correct (be that a straight .items() or using six).